src/main/java/br/com/sbk/sbking/dto/LobbyScreenTableDTO.java

```diff
@@ -1,29 +1,42 @@
 package br.com.sbk.sbking.dto;
 
 import java.util.Map;
+import java.util.UUID;
 
 import br.com.sbk.sbking.core.Direction;
 import br.com.sbk.sbking.core.Player;
-import br.com.sbk.sbking.core.TrickGame;
+import br.com.sbk.sbking.networking.server.Table;
 
 public class LobbyScreenTableDTO {
 
-    private final Map<Direction, Player> playersDirection;
-    private final TrickGame trickGame;
-    private final int numberOfSpectators;
+    private UUID id;
```

Comment:
Why you removed the final?

Reply:
kryo cannot serialize them if they are final :(